bin/ns_unshared

```diff
@@ -8,10 +8,8 @@ require 'fileutils'
   '/proc/sys/net/bridge/bridge-nf-call-arptables',
   '/proc/sys/net/bridge/bridge-nf-call-ip6tables',
   '/proc/sys/net/bridge/bridge-nf-call-iptables'
-].each do
-  File.write _1, '0'
-rescue Errno::ENOENT
-  # br_netfilter is not loaded
+].each do |file|
+  File.write file, '0' if File.exist? file
```

Comment:
Replacing `Errno::ENOENT` rescue with `File.exist?` check is not good as it is possible to remove the file after `File.exist?` check while such a failure is not possible with `Errno::ENOENT` rescue. Though such a failure is unlikely to happen in this particular code, the sequence of file opening after checking should be considered as a common anti-pattern in UNIX programming and avoided.

Reply:
The latest rubocop ask me to remove `rescue`. I will think about how to rewrite this part.

Reply:
Weird. It seems not complained about an empty `rescue` in `lib/setupmanagers/qemuhck/qemu_machine.rb`.

Reply:
It seems that an offense of `Style/NumberedParameters` somewhat triggers `Lint/SuppressedException`. Fixing `Style/NumberedParameters` also suppresses `Lint/SuppressedException`. Likely a bug of RuboCop.

Reply:
Opened https://github.com/rubocop/rubocop/pull/11151.